.travis.yml

```diff
@@ -1,22 +1,90 @@
-dist: bionic
-
 language: cpp
 
-compiler:
-  - gcc
-
-addons:
-  apt:
-    packages:
-      - g++-8
+jobs:
+  include:
+   - stage: "Test"
+     name: "Ubuntu Trusty"
+     os: linux
+     dist: trusty
+     addons:
+       apt:
+         sources:
+         - ubuntu-toolchain-r-test
+         packages:
+         - g++-9
+         - gcc-9
+   - stage: "Test"
+     name: "Ubuntu Xenial"
+     os: linux
+     dist: xenial
+     addons:
+       apt:
+         sources:
+         - ubuntu-toolchain-r-test
+         packages:
+         - g++-9
+         - gcc-9
+#   - stage: "Test"
+#     name: "Ubuntu Bionic"
+#     os: linux
+#     dist: bionic
+#     addons:
+#       apt:
+#         sources:
+#         - ubuntu-toolchain-r-test
+#         packages:
+#         - g++-9
+#         - gcc-9
 
 before_install:
-  - wget https://github.com/bazelbuild/bazel/releases/download/1.1.0/bazel_1.1.0-linux-x86_64.deb
-  - sudo dpkg -i bazel_1.1.0-linux-x86_64.deb
+  - sudo dpkg --list | grep gcc
+  - sudo ln -sf /usr/bin/gcc-9 /usr/bin/gcc
+  - sudo ln -sf /usr/bin/g++-9 /usr/bin/g++
+  - gcc --version || true
+  - wget https://github.com/bazelbuild/bazel/releases/download/1.2.0/bazel_1.2.0-linux-x86_64.deb
+  - sudo dpkg -i bazel_1.2.0-linux-x86_64.deb || true
+  - sudo apt-get -f install
+  - gcc --version || true
+  - bazel --version
 
 script:
   - bazel build --noshow_progress --cxxopt='-std=c++17' //...
```

Comment:
The c++17 should now (after ca3fce403ab5bf2cf56795b747da718c2be2b3ca )  work with C++11